### PR TITLE
Less obnoxious domain block message

### PR DESCRIPTION
**Pull request by ghost**
Intended to address #61

This looks to be a fairly straightforward change in the English language translation file. Took me a while to find it though, so this was indeed a good first issue. :+1:

### CODE REVIEW

**Review by 1011X — DISMISSED** [review]

File: app/javascript/mastodon/locales/en.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
   "confirmations.delete_list.confirm": "Delete",
   "confirmations.delete_list.message": "Are you sure you want to permanently delete this list?",
   "confirmations.domain_block.confirm": "Hide entire domain",
-  "confirmations.domain_block.message": "Are you really, really sure you want to block the entire {domain}? In most cases a few targeted blocks or mutes are sufficient and preferable. You will not see content from that domain in any public timelines or your notifications. Your followers from that domain will be removed.",
+  "confirmations.domain_block.message": "Blocking means you will not see content from {domain} in any public timelines nor your notifications. Your followers from that domain will be removed.",
```
Comment (clarfonthey) [inline app/javascript/mastodon/locales/en.json]: This should be neither...nor or not...or. A mix of both feels a bit strange to me.

**Review by mal0ki — CHANGES_REQUESTED** [review]
"Your followers from that domain will be removed." Maybe change this specifically to "Any followers from that domain (or {domain}) will be removed"

**Comment by ghost** [comment]
Updated as per @mal0ki feedback :)

btw is this something translation team needs to keep an eye on? I'm only making the change for the English language locale, but presumably other locales will have foreign language versions of the old text.

**Comment by mal0ki** [comment]
> 
> 
> Updated as per @mal0ki feedback :)
> 
the (or {domain} ) was supposed to be "pick this or the other option in your edit" :D Not to be included.

**Comment by ghost** [comment]
Damn it @mal0ki! :stuck_out_tongue_closed_eyes: 

I've now amended to "Blocking means you will not see content from {domain} in any public timelines nor your notifications. Any followers from {domain} will also be removed."

**Review by mal0ki — DISMISSED** [review]

**Comment by clarfonthey** [comment]
Not should be neither, or nor should be or... right now the halfness looks weird.

**Comment by ghost** [comment]
@clarfon I agree. Technically it's not wrong, but most are used to seeing "nor" with "neither". I went back and forth on it myself, as I liked @ryliejamesthomas suggestion in #61 

Have now fixed it to say "or" instead.

**Review by clarfonthey — APPROVED** [review]

**Comment by ryliejamesthomas** [comment]
Yeah, no worries. I think 'neither' is something we just don't say in Australia?, and I do quite like awkward-sounding-but-correct English, but no problems here with making changes :)

**Comment by clarfonthey** [comment]
Honestly, "neither...nor" seems grammatically excessive nowadays. it's like saying "the things up with which I put" instead of "the things I put up with."

**Review by 1011X — APPROVED** [review]

**Pull request merged**